scripts/variantstore/wdl/extract/utils.py

```diff
@@ -53,3 +53,22 @@ def execute_with_retry(client, label, sql):
 
 def utf8len(s):
     return len(s.encode('utf-8'))
+
+
+def write_job_stats(jobs, client, fq_dataset, call_set_identifier, step, call, shard_identifier):
+    total = 0
+
+    for query_return in jobs:
+        job = client.get_job(query_return['job'])
+        bytes_billed = int(0 if job.total_bytes_billed is None else job.total_bytes_billed)
+        total = total + bytes_billed
+        print(query_return['label'], "-- jobid:  (", job.job_id, ") <====> Cache Hit:", job.cache_hit, bytes_billed/(1024 * 1024), " MiBs")
+    print("\nTotal GiBs billed ", total/(1024 * 1024 * 1024), " GiBs\n")
+
+    # populate cost_observability data
+    sql = f"""INSERT INTO `{fq_dataset}.cost_observability`
+            (call_set_identifier, step, call, shard_identifier, event_key, call_start_timestamp, event_timestamp, event_bytes)
+            VALUES('{call_set_identifier}', '{step}', '{call}', '{shard_identifier}', 'BigQuery Query Billed',
```

Comment:
is this supposed to be '[BigQuery Query Scanned](https://github.com/broadinstitute/gatk/blob/55398f578abfd76322065a15459070683fa5cc90/src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/ExtractFeatures.java#L195)' like in George's PR?